### ISSUE TITLE
Options Don't Seem to be Working

### ISSUE THREAD

**Issue by barkins** [fancyapps/fancybox#1543]
Not sure if I'm doing something wrong, but including the following option or any other option doesn't seem to do anything. All the defaults remain working. 

```
(function($){

  $(document).ready(function(){

    $('.lightbox').fancybox({
       animationEffect : false,
    });

  });

})(jQuery)
```

**Comment by fancyapps** [comment]
Hi,

Without seeing your live page, it is hard to tell. 
1) Make sure that you include only one fancybox JS file (and only once)
2) Open developer tools and check console if there are any JS error 

https://codepen.io/anon/pen/aybpKP?editors=1010

**Comment by barkins** [comment]
Thanks for the example. Here's the code I'm working with: https://codepen.io/barkins/pen/da948fdfd906acda2921db4c9fa59b3c

Appreciate any insight.

**Comment by fancyapps** [comment]
Hi,

You have to upgrade fancyBox to the latest version, because options like `animationEffect` are available starting from v3.1
